app/namespaces/data_provider/data_provider_controller.py

```diff
@@ -40,18 +40,25 @@ def post(self):
         columns = data.get('columns')
         research_fields = data.get('research_fields')
         prioritize_estimates = data.get('prioritize_estimates', True)
-        start_date, end_date = convert_start_end_dates(data)
-
-        result = get_filtered_records(research_fields, filters, columns, start_date=start_date, end_date=end_date,
+        sampling_start_date, sampling_end_date = convert_start_end_dates(data, use_sampling_date=True)
+        publication_start_date, publication_end_date = convert_start_end_dates(data, use_sampling_date=False)
+
+        result = get_filtered_records(research_fields,
+                                      filters,
+                                      columns,
+                                      sampling_start_date=sampling_start_date,
+                                      sampling_end_date=sampling_end_date,
+                                      publication_start_date=publication_start_date,
```

Comment:
@dtanying - note the addition of publication start date and end date filters here.
You'll want to provide a Jan 1 2020 publication start date and Dec 31 2020 publication end date when you are making requests, to ensure that only data that meets the inclusion criteria for the systematic review is included